### ISSUE TITLE
In the Product detail Page, can we add a video in the thumbnails ?

### ISSUE THREAD

**Issue by mksent** [Shopify/starter-theme#180]
In the Product detail Page, can we add a video in the thumbnails?
![Thumbnails](https://user-images.githubusercontent.com/70639698/91976300-594a0580-ed3e-11ea-9da0-75d2ec28fbc2.png)
?

**Comment by Cam** [comment]
Yes, you can use {{ media.preview_image }} as your image source if you are using video in media files.
